src/prefect/agent/local/agent.py

```diff
@@ -78,9 +79,13 @@ def deploy_flows(self, flow_runs: list) -> None:
 
             storage = StorageSchema().load(flow_run.flow.storage)
             if not isinstance(StorageSchema().load(flow_run.flow.storage), Docker):
-                self.logger.error(
-                    "Storage for flow run {} is not of type Docker.".format(flow_run.id)
+                msg = "Storage for flow run {} is not of type Docker.".format(
```

Comment:
Could you add something noting that this log is coming from an Agent

Reply:
When you say "log" do you mean the state message?

Reply:
Ah yep that is in fact what I meant

Reply:
Updated